### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by SincerelyBrittany**
Hi @shivaylamba & @blobbyblobfish, I updated the readme for our repo. Please review and add your contact information to the bottom of the readme. I added my twitter but feel free to add you linkedin or anything else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,133 @@
...
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+## Contact
+Connect/Follow us:
+
+Brittany:
+[![Twitter][twitter-shield]][brittany-twitter-url]
+
+Alice:
```
Comment (blobbyblobfish) [inline README.md:109]:
```suggestion
[![LinkedIn][linkedin-shield](https://linkedin.com/in/aliceyunwang)]
```

Comment (blobbyblobfish) [reply README.md:109]: @SincerelyBrittany I added my LinkedIn. Could you let me know if this syntax works?

File: README.md [hunk 2]
```diff
@@ -1 +1,133 @@
...
+[contributors-url]: https://github.com/blobbyblobfish/piggy-bank/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/blobbyblobfish/piggy-bank.svg?style=flat-square
+[forks-url]: https://github.com/blobbyblobfish/piggy-bank/network/members
+[stars-shield]: https://img.shields.io/github/stars/blobbyblobfish/piggy-bank.svg?style=flat-square
+[stars-url]: https://github.com/blobbyblobfish/piggy-bank/stargazers
+[issues-shield]: https://img.shields.io/github/issues/blobbyblobfish/piggy-bank.svg?style=flat-square
+[issues-url]: https://github.com/blobbyblobfish/piggy-bank/issues
+[license-shield]: https://img.shields.io/github/license/blobbyblobfish/piggy-bank.svg?style=flat-square
+[license-url]: https://github.com/blobbyblobfish/piggy-bank/blob/readme/LICENSE
+<!-- [linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat-square&logo=linkedin&colorB=555 -->
```
Comment (blobbyblobfish) [inline README.md:130]:
```suggestion
[linkedin-shield]: https://img.shields.io/badge/-LinkedIn-black.svg?style=flat-square&logo=linkedin&colorB=555
```

**Comment by blobbyblobfish** [comment]
I like this Readme template, did you make it or find it somewhere?

**Review by shivaylamba — COMMENTED** [review]
Looks Good

**Pull request merged**